src/model/window-guardian.ts

```diff
@@ -175,6 +175,10 @@ export const makeWindowGuardian = (
             cssIDs,
             data: {
                 ...dcrDocumentData,
+                NAV: {
+                    subNavSections: dcrDocumentData.NAV.subNavSections,
+                    currentNavLink: dcrDocumentData.NAV.currentNavLink,
+                },
```

Comment:
As above, we've removed the need to pass the 'NAV' object so these properties can just be put on CAPI?

Reply:
I think using the same approach we used to support NoJS we could apply this to the Subnav, I do not think it belongs in CAPI